webapp/components/general/ControlPanelTreeItem.js

```diff
@@ -65,7 +64,7 @@ const useStyles = makeStyles((theme) => ({
   colorPicker: {
     position: 'absolute',
     zIndex: 1000,
-    right: 0,
+    right: '-2.7rem',
```

Comment:
@aranega great job in general, one small note.
This negative right positioning is balancing the colorPickerBox (that wraps the colorPicker class), so if you zero [this](https://github.com/MetaCell/NetPyNE-UI/blob/2f824c84c2edb1eddf9db022f82973b6f6aa7cfc/webapp/components/general/ControlPanelTreeItem.js#L57) you don't need this negative value but this can be set to 0 as well, other than this minor looks pretty good to me :)
Thanks!

Reply:
@ddelpiano Thanks so much for the review and for pointing out where I could modify better the code! I put `-2.7rem` really using a wet-finger approach (I put what was looking the best in the screen), I wouldn't have think I was going against the exact same measure somewhere else :smile:. I'm correcting this right away!